modules/core/zio/src/weaver/ziocompat/Live.scala

```diff
@@ -0,0 +1,24 @@
+package weaver
+package ziocompat
+
+import zio.{ IO, URLayer, ZEnv, ZIO }
+
+/**
+ * Service used for getting the real environment during tests. This is useful
+ * for timing or getting random values during tests. For example, getting a
+ * random port for starting servers or timing an action with the real clock.
+ * This pattern is inspired by ZIO-test
+ */
+object Live {
```

Comment:
This seems like a pretty overloaded term in zio-land. Any alternative name you can think of ?